### PR TITLE
Adding email field in Tenant Model

### PR DESCRIPTION
**Pull request by TonioGela**
This PR depends on https://github.com/pagopa/interop-be-tenant-management/pull/43

### CODE REVIEW

**Review by galales — APPROVED** [review]

**Review by beetlecrunch — APPROVED** [review]

**Review by beetlecrunch — APPROVED** [review]

File: src/main/scala/it/pagopa/interop/tenantprocess/api/adapters/TenantManagementAdapters.scala [hunk 1]
```diff
@@ -34,10 +37,22 @@ object TenantManagementAdapters extends SprayJsonSupport with DefaultJsonProtoco
...
       createdAt = t.createdAt,
-      updatedAt = t.updatedAt
+      updatedAt = t.updatedAt,
+      mails = t.mails.map(_.toApi)
     )
   }
 
+  implicit class DependencyMailWrapper(private val m: DependencyMail) extends AnyVal {
+    def toApi: Mail                = Mail(kind = m.kind.toApi, address = m.address, createdAt = m.createdAt)
+    def toSeed: DependencyMailSeed = DependencyMailSeed(m.kind, m.address)
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/tenantprocess/api/adapters/TenantManagementAdapters.scala]: aren't these missing the description?

**Review by galales — COMMENTED** [review]
I guess also here the description should be set to mandatory

**Comment by TonioGela** [comment]
> I guess also here the description should be set to mandatory

Nope, @ruggerocastagnola dixit:

```
Specifica API

Il body deve contenere due campi:
- un campo stringa per l'inserimento della mail, required
- un campo stringa per l'inserimento di una descrizione da associare alla mail, max. 250 caratteri, opzionale
```

File: src/main/scala/it/pagopa/interop/tenantprocess/api/adapters/TenantManagementAdapters.scala (hunk 1 above)
Comment (TonioGela) [reply src/main/scala/it/pagopa/interop/tenantprocess/api/adapters/TenantManagementAdapters.scala]: sharp eye, thanks

**Comment by TonioGela** [comment]
> 🎉 All dependencies have been resolved !

Ma te che voi?

**Pull request merged**